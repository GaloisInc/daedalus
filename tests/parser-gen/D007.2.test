D007.ddl
--interp=inputs/D007.2.inp
--gen