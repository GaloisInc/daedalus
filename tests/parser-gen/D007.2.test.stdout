[WARNING] ./D007.ddl:12:10--15:5:
            Using unbiased choice may be inefficient.
context:
  "Main"
input: File context:
Length: 7 (0x7) bytes
0000:   67 67 67 67  61 64 [1;91m0a[0m                     ggggad[1;91m.[0m

